go.mod

```diff
@@ -1,17 +1,28 @@
 module github.com/VictoriaMetrics/VictoriaLogs
 
-go 1.14
+go 1.17
 
 require (
-	github.com/VictoriaMetrics/VictoriaMetrics v1.44.0-cluster.0.20201020114246-0db7c2b50094
-	github.com/VictoriaMetrics/fastcache v1.5.7
-	github.com/VictoriaMetrics/metrics v1.12.3
-	github.com/cespare/xxhash/v2 v2.1.1
-	github.com/gogo/protobuf v1.3.1
-	github.com/golang/snappy v0.0.2
+	github.com/VictoriaMetrics/VictoriaMetrics v1.65.0-cluster
```

Comment:
When vm has not yet released the next version we can rely on a latest commit hash as the version. You can try `go get github.com/VictoriaMetrics/VictoriaMetrics@2b19dd86010f6ccd78a04d20d257ebd32b5f21ac`